tests/integrated/test-code-style/runtest

```diff
@@ -15,6 +15,38 @@ grep [\(,][[:space:]]*const[[:space:]][[:alnum:]]*[[:space:]][[:alnum:]]*[[:spac
     echo "Declaration of functions shouldn't declare pass-by-value as const"
 
 
+GIT=git
+if $GIT status &>/dev/null
+then
+    # Find autogenerated files
+    autogens=$(for f in $(find $paths |grep '\.in\.[a-z0-9A-Z]*$')
```

Comment:
This doesn't appear to work -- there's no `BOUT_TOP` in here, and you'd need to loop over `$paths`. The test passes though.

Reply:
Should be fixed now